sbi/inference/base.py

```diff
@@ -63,6 +65,10 @@ def infer(
         num_simulations: Number of simulation calls. More simulations means a longer
             runtime, but a better posterior estimate.
         num_workers: Number of parallel workers to use for simulations.
+        init_kwargs: Additional keyword arguments as dict for the inference method
+            which are passed to `__init__`.
+        call_kwargs: Additional keyword arguments as dict for the call to
```

Comment:
we `call`? Why not `build_posterior_kwargs`?

Reply:
This has also led to some confusion for me. `build_posterior` makes a lot more sense.